src/resources/views/base/auth/passwords/email.blade.php

```diff
@@ -2,7 +2,7 @@
 
 {{-- Main Content --}}
 @section('content')
-    <div class="row justify-content-center">
+    <div class="row justify-content-center align-items-center d-flex flex-row min-vh-100">
```

Comment:
What if instead of using helper classes here... we 
1. Add some general-purpose classes to this blade view... (eg. `backpack-reset-password-card`), which is a change we can do on the `v6` branch directly;
2. Then inside the CSS file for this theme, we target that class, to add the css rules we want;

That should keep the changes we make to the blade files GENERAL, instead of theme-specific.